### ISSUE TITLE
Code pickaxes for all ores

### ISSUE THREAD

**Issue by Inshomothal** [RedstoneChicken/Krafted#13]
Some pickaxes wont interact with ores from other addons. We can try and code that into them so the addon pack works together.

**Comment by ZhyuroKun** [comment]
Agree, we have to note every new ores then insert them in to the pickaxe which is a big time consuming

**Comment by ZhyuroKun** [comment]
Actually, what about new stones?
Like the ones from cavern addon
